source/help/messaging/sending-messages.rst

```diff
@@ -0,0 +1,60 @@
+.. _sending-messages:
+
+Messaging Basics
+================
+
+After a message has been sent, you or others can reply to it and link to it. Depending on which edition of Mattermost you have and how it's configured by the System Admin, you can also edit and delete your own messages.
+
+Message Types
+-------------
+
+Mattermost has two types of messages: posts and replies.
+
+Posts
+  Posts can be considered parent messages. They are the messages that often start a thread of replies. Posts can be written and sent from the text input box at the bottom of the center pane.
+
+Replies
+  Reply to posts to keep conversations organized in threads.
+
+  Reply to a post by clicking the reply icon next to any message text. This action opens the right-hand sidebar (RHS) where you can see the message thread, then compose and send your reply. Replies are indented slightly in the center pane to indicate that they are child messages of a parent post.
+
+  When composing a reply in the right-hand sidebar, click the expand/collapse icon with two arrows at the top of the sidebar to make things easier to read.
+
+Sending a Message
+-----------------
+
+Write a message by typing into the text input box, then press **ENTER** to send it. Use **SHIFT+ENTER** to create a new line without sending a message. To send messages by pressing **CTRL+ENTER**, go to **Main Menu > Account Settings > Send messages on Ctrl + Enter**.
+
+Editing a Message
+-----------------
+
+The ability to edit a message after it is sent depends on which edition you are using.
+
+Team Edition
```

Comment:
End users won't understand the term "Enterprise edition". Propose we can just say:

"Edit a message by clicking the **[...]** icon next to any message that you’ve sent, then click **Edit**. When you're done editing, press **ENTER** to save the changes. Message editing may be time limited or disabled by your System Admin.

Note: Edited messages do not trigger new mention notifications, desktop notifications, or notification sounds. "